test/src/ui/js/test_Game.js

```diff
@@ -1671,28 +1673,101 @@ test("test_Game.createGameMatDieDiv", function(assert) {
 });
 
+test("test_Game.backgroundImagePath", function(assert) {
+  Api.game = { player: { dieBackgroundType: 'symmetric' } };
+
+  assert.ok(Game.backgroundImagePath(1, 'active', true, 'symmetric').indexOf('images/die/symmetric/d2active.png') >= 0, 'active d1');
+  assert.ok(Game.backgroundImagePath(2, 'active', true, 'symmetric').indexOf('images/die/symmetric/d2active.png') >= 0, 'active d2');
+  assert.ok(Game.backgroundImagePath(3, 'active', true, 'symmetric').indexOf('images/die/symmetric/d4active.png') >= 0, 'active d3');
+  assert.ok(Game.backgroundImagePath(4, 'active', true, 'symmetric').indexOf('images/die/symmetric/d4active.png') >= 0, 'active d4');
+  assert.ok(Game.backgroundImagePath(5, 'active', true, 'symmetric').indexOf('images/die/symmetric/d6active.png') >= 0, 'active d5');
+  assert.ok(Game.backgroundImagePath(6, 'active', true, 'symmetric').indexOf('images/die/symmetric/d6active.png') >= 0, 'active d6');
+  assert.ok(Game.backgroundImagePath(7, 'active', true, 'symmetric').indexOf('images/die/symmetric/d8active.png') >= 0, 'active d7');
+  assert.ok(Game.backgroundImagePath(8, 'active', true, 'symmetric').indexOf('images/die/symmetric/d8active.png') >= 0, 'active d8');
+  assert.ok(Game.backgroundImagePath(9, 'active', true, 'symmetric').indexOf('images/die/symmetric/d10active.png') >= 0, 'active d9');
+
+  Api.game.player.dieBackgroundType = 'realistic';
+
+  assert.ok(Game.backgroundImagePath(10, 'active', true, 'realistic').indexOf('images/die/realistic/d10active.png') >= 0, 'active d10');
+  assert.ok(Game.backgroundImagePath(11, 'active', true, 'realistic').indexOf('images/die/realistic/d12active.png') >= 0, 'active d11');
+  assert.ok(Game.backgroundImagePath(12, 'active', true, 'realistic').indexOf('images/die/realistic/d12active.png') >= 0, 'active d12');
+  assert.ok(Game.backgroundImagePath(13, 'active', true, 'realistic').indexOf('images/die/realistic/d20active.png') >= 0, 'active d13');
+  assert.ok(Game.backgroundImagePath(19, 'active', true, 'realistic').indexOf('images/die/realistic/d20active.png') >= 0, 'active d19');
+  assert.ok(Game.backgroundImagePath(20, 'active', true, 'realistic').indexOf('images/die/realistic/d20active.png') >= 0, 'active d20');
+  assert.ok(Game.backgroundImagePath(21, 'active', true, 'realistic').indexOf('images/die/realistic/d30active.png') >= 0, 'active d21');
+  assert.ok(Game.backgroundImagePath(29, 'active', true, 'realistic').indexOf('images/die/realistic/d30active.png') >= 0, 'active d29');
+  assert.ok(Game.backgroundImagePath(30, 'active', true, 'realistic').indexOf('images/die/realistic/d30active.png') >= 0, 'active d30');
+  assert.ok(Game.backgroundImagePath(31, 'active', true, 'realistic').indexOf('images/die/realistic/d30active.png') >= 0, 'active d31');
+
+  Api.game.player.dieBackgroundType = 'symmetric';
+
+  assert.ok(Game.backgroundImagePath(1, 'active', false, 'symmetric').indexOf('images/die/symmetric/d2inactive.png') >= 0, 'inactive d1');
```

Comment:
No, I think this is correct. As far as I understand, the status is either 'active' or 'taken'. The thing that distinguishes 'active' from 'inactive' is whether it's clickable or not, thus the different third parameter.